### ISSUE TITLE
what is the range of optical flow value in an image? 

### ISSUE THREAD

**Issue by Pixie8888** [sniklaus/pytorch-pwc#50]
Hi, 
Thanks for this work! I use this pwcnet to estimate optical flow. I have a question regarding the value range of optical flow. Is it range from (-w+1, w-1) (-h+1, h-1)? If I want to normalise optical flow to (-1,1), should I follow this:
optical_flow[0,:,:] = optical_flow[0,:,:]/ (w-1)
optical_flow[1,:,:] = optical_flow[1,:,:]/ (h-1)?
(w and h is the image size. optical_flow_shape = (2, h, w))

**Comment by sniklaus** [comment]
The neural network is not constrained to any range. If you want the estimated optical flow to be within the range of [-1, 1] you will have to divide by `w` and `h` respectively.
